broker-daemon/broker-rpc/wallet-service/release-channels.js

```diff
@@ -0,0 +1,43 @@
+const { PublicError } = require('grpc-methods')
+
+/**
+ * Grabs public lightning network information from relayer and opens a channel
+ *
+ * @param {Object} request - request object
+ * @param {Object} request.params
+ * @param {RelayerClient} request.relayer
+ * @param {Logger} request.logger
+ * @param {Engine} request.engines
+ * @param {Object} responses
+ * @param {function} responses.EmptyResponse
+ * @return {responses.EmptyResponse}
+ */
+async function releaseChannels ({ params, relayer, logger, engines, orderbooks }, { ReleaseChannelsResponse }) {
+  const { market } = params
+  const orderbook = orderbooks.get(market)
+
+  if (!orderbook) {
+    throw new PublicError(`${market} is not being tracked as a market.`)
+  }
+  const [ baseSymbol, counterSymbol ] = market.split('/')
```

Comment:
@martineehrlich is `engines.get` case sensitive? We may have to upcase `baseSymbol` and `counterSymbol` as param checking?

Reply:
@martineehrlich we decided that if the user provides the incorrect currency symbol that these errors are sufficient for now